chefs_soils.py

```diff
@@ -1,73 +1,1379 @@
+  name = SOURCE_SITE_USE_NAME_DIC.get(id)
+  if name is not None:
+    return name
+  else:
+    return id
+
+def convert_receiving_site_use_to_name(id):
+  name = RECEIVING_SITE_USE_NAME_DIC.get(id)
+  if name is not None:
+    return name
+  else:
+    return id
+
+def convert_soil_quality_to_name(id):
+  name = SOIL_QUALITY_NAME_DIC.get(id)
+  if name is not None:
+    return name
+  else:
+    return id
+
+def convert_land_ownership_to_name(id):
+  name = LAND_OWNERSHIP_NAME_DIC.get(id)
+  if name is not None:
+    return name
+  else:
+    return id
+
+def create_popup_link_body(_site_dic):
+  _link_body = ''
+
+  if _site_dic['SID'] is not None and _site_dic['SID'].strip() != '':
+    _link_body = urllib.parse.quote(_site_dic['SID']) #Site ID
+  elif _site_dic['PID'] is not None and _site_dic['PID'].strip() != '':
+    _link_body = urllib.parse.quote(_site_dic['PID']) #PID
+  elif _site_dic['PIN'] is not None and _site_dic['PIN'].strip() != '':
+    _link_body = urllib.parse.quote(_site_dic['PIN']) #PIN
+  elif _site_dic['latitude'] is not None and _site_dic['longitude'] is not None:
+    _link_body = urllib.parse.quote(str(_site_dic['latitude'])+','+str(_site_dic['latitude'])) #Site lat/lon
+  elif _site_dic['ownerAddress'] is not None and _site_dic['ownerAddress'].strip() != '':
+    _link_body = urllib.parse.quote(_site_dic['ownerAddress']) #Site Owner Address
+  elif _site_dic['ownerCompany'] is not None and _site_dic['ownerCompany'].strip() != '':
+    _link_body = urllib.parse.quote(_site_dic['ownerCompany'])  #Site Owner Company  
+
+  return _link_body
+
+# create links to popup on Map using receiving sites searching keywords
+def create_popup_links(sites, site_type):
+  _popup_links = '' 
+
+  if sites is not None:
+    for _site_dic in sites:
+      _link_body = create_popup_link_body(_site_dic)
+
+      # create popup link
+      if _link_body != '' and (site_type  == 'SR' or site_type  == 'HV'): #SR: for Soil Relocation Notication, HV: High Volume Receiving Notification
+        _link = '<a href=' + WEBMAP_POPUP_URL + '?id=' + WEB_MAP_APP_ID + '&find=' 
+        _link += _link_body
+        if site_type  == 'SR':
+          _link += '>Link to new submission</a><br/>'
+        elif site_type  == 'HV':
+          _link += '>Link to new high volume receiving site registration</a><br/>'
+
+      _popup_links += _link
+  return _popup_links
+
+def create_land_file_numbers(cefs_dic, field):
+  _land_file_numbers = []
+  if cefs_dic.get(field) is not None : 
+    for _item in cefs_dic[field]:
+      for _v in _item.values():
+        if _v != '':
+          _land_file_numbers.append(_v)
+
+  if len(_land_file_numbers) > 0: 
+    _land_file_numbers = "\"" + ",".join(_land_file_numbers) + "\""   # could be more than one    
+
+  return _land_file_numbers if len(_land_file_numbers) > 0 else None
+
+def create_receiving_site_lan_uses(cefs_dic, field):
+  _land_uses = []
+  for _k, _v in cefs_dic[field].items():
+    if _v == True:
```

Comment:
should the `==` be replaced by `is`?

Reply:
I found this Python PE8 style guideline. So the comparing with Boolean was changed to following the guideline (e.g "if greeting:")

https://peps.python.org/pep-0008/#programming-recommendations

![image](https://user-images.githubusercontent.com/107894308/195915404-dd91e4ff-133b-4261-8a19-9a800910b96e.png)